check-mysql/lib/check-mysql.go

```diff
@@ -19,6 +22,13 @@ type mysqlSetting struct {
 	Pass   string `short:"P" long:"password" default:"" description:"Password" env:"MYSQL_PASSWORD"`
 }
 
+type mysqlVersion struct {
+	major     int
+	minor     int
+	patch     int
+	stability string
```

Comment:
Is that really means stability?

For example in MySQL 5.5 in Debian, I got following result:
```
mysql> show variables like 'version';
+---------------+---------------------+
| Variable_name | Value               |
+---------------+---------------------+
| version       | 5.5.44-0+deb8u1-log |
+---------------+---------------------+
1 row in set (0.00 sec)
```

Apparently `deb8u1` does not indicate stability.
I guess this is just set by `-DMYSQL_SERVER_SUFFIX` option on build : [ref](https://salsa.debian.org/mariadb-team/mysql/-/blob/mysql-8.0/debian/master/debian/rules#L94). Is it just be treated as "suffix", I think.

Reply:
I will delete the field since this is not used from check-mysql.

Reason for the name `stability` is below, as an aside.

---

MySQL doc says that release versions is composed of three numbers and an optional suffix, and the suffix is described as `stability level of the release`.

> Release names can also include a suffix to indicate the stability level of the release. Releases within a series progress through a set of suffixes to indicate how the stability level improves. The possible suffixes are: ...
> 
> https://dev.mysql.com/doc/refman/8.0/en/which-version.html